components/log-viewer-webui/server/eslint.config.mjs

```diff
@@ -25,13 +25,31 @@ const EslintConfig = [
                         "Type.Enum",
                         "Type.Integer",
                         "Type.Literal",
+                        "Type.Null",
                         "Type.Required",
                         "Type.Union",
                         "Value.Errors",
                         "Value.Parse",
                     ],
                 },
             ],
+            // eslint-disable-next-line no-warning-comments
+            // TODO: update eslint-config-yscope
+            "@stylistic/max-len": [
+                "warn",
+                {
+                    code: 100,
+                    comments: 100,
+                    ignoreComments: false,
+                    ignorePattern: "^(import\\s.+\\sfrom\\s.+|\\} from)",
+                    ignoreRegExpLiterals: true,
+                    ignoreStrings: false,
+                    ignoreTemplateLiterals: false,
+                    ignoreTrailingComments: false,
+                    ignoreUrls: true,
+                    tabWidth: 4,
+                },
+            ],
```

Comment:
i suppose this is added to ignore a super long import line? is it still needed in current code?